doc/users/prev_whats_new/whats_new_1.3.rst

```diff
@@ -304,9 +304,9 @@ rcParam has been set, and will not retroactively affect already
 existing text objects.  This brings their behavior in line with most
 other rcParams.
 
-``savefig.jpeg_quality`` added to rcParams
-``````````````````````````````````````````
-rcParam value ``savefig.jpeg_quality`` was added so that the user can
+:rc:`savefig.jpeg_quality` added to rcParams
```

Comment:
Wouldn't do that in a heading. It's quite verbose due to mentioning the default.

https://9880-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/users/prev_whats_new/whats_new_1.3.html#rcparams-savefig-jpeg-quality-default-95-added-to-rcparams

Reply:
What about
```
Added :rc:`savefig.quality`
```
?
I think the title reads well in that case ("Added rcParams["savefig.quality"] (default: 95)").

Reply:
Ok by me.

Reply:
done